bin/crew

```diff
@@ -1126,24 +1126,27 @@ def install_package(pkgdir)
         end
       end
     end
-    if File.exist?("#{CREW_PREFIX}/bin/rdfind")
-        puts "Using rdfind to convert duplicate files to hard links."
-        system "#{CREW_PREFIX}/bin/rdfind -removeidentinode true -makesymlinks false -makehardlinks true -makeresultsfile false ."
+    if system 'command -v rdfind', out: '/dev/null'
```

Comment:
Why not Rubyize?

Reply:
Oh, I think you mean using the previous `File.exist?` one. `command -v` can check in the whole $PATH, why not :)

Reply:
No Ruby way to do the same thing?  Is rsync available oob?  If not, the only place it would be located is at `#{CREW_PREFIX}/bin`, right?

Reply:
The reason for search it in whole PATH is to let users can specify and use their executable in other locations

Reply:
If users are relocating executables, they should be using `CREW_PREFIX` to do it.

Reply:
Change reverted